GitUI/ScriptsEngine/IScriptHostControl.cs

```diff
@@ -7,5 +7,34 @@ public interface IScriptHostControl
         GitRevision? GetLatestSelectedRevision();
         IReadOnlyList<GitRevision> GetSelectedRevisions();
         Point GetQuickItemSelectorLocation();
+
+        void GoToRef(string? refName, bool showNoRevisionMsg, bool toggleSelection = false);
```

Comment:
❓ Use `GitUICommands.BrowseGoToRef` instead? Might need to be added to `IGitUICommands` yet.

Reply:
From what I can tell, the difference is
```csharp
            using (WaitCursorScope.Enter())
```
Since it calls through `FormBrowse`, rather than calling `RevisionGrid.GoToRef` directly.

What makes me doubly unsure is the fact that there's `GitUICommands.GetSelectedRevisions` as well. Accessing it directly like that would bring `IScriptHostControl`'s purpose into question.

Reply:
I think a wait cursor is not very important while selecting a revision.

> What makes me doubly unsure is the fact that there's `GitUICommands.GetSelectedRevisions` as well. Accessing it directly like that would bring `IScriptHostControl`'s purpose into question.

👍
And `IsRevisionGrid / HasSelectedRevision` is equivalent to `UICommands.BrowseRepo is not null`.

@RussKie, shall we go ahead using GitUICommands instead of dedicated `IScriptHostControl` members?

Reply:
The more I think about it, I'm thinking getting things _out_ of `IScriptHostControl`, while it may or may not have value, is probably beyond the scope (and against the flow) of this PR. While it's theoretically possible to get all of `IScriptHostControl` merged into `IGitUICommands`, that is something that needs to be considered separately.

For the sake of this PR, it's better to keep things separate, thereby keeping behavior as close to what it currently is as possible.